beavis-ci/beavis-ci.sh

```diff
@@ -37,6 +38,9 @@ html=0
 all=0
 src="$0"
 branch='master'
+repo='LSSTDESC/DC2-analysis'
+jupyter='/usr/common/software/python/3.6-anaconda-4.4/bin/jupyter'
```

Comment:
I think we'll need a `-j` option to allow the user to select their own `jupyter` path, or use `which jupyter` to make `beavis-ci` system agnostic.